tests/test_wfirst.py

```diff
@@ -37,107 +37,50 @@
 
 @timer
 def test_wfirst_wcs():
-    """Test the WFIRST WCS routines against those from software provided by WFIRST project office.
+    """Test the WFIRST WCS routines against the one produced by code from Chris Hirata.
```

Comment:
I wonder if it would be worth keeping the old test in here, but disabling it with a comment that we need to get a new updated WCS from the WFIRST project office to compare against.  (e.g. rename it `dont_test_wfirst_wcs`)  Then give your current test here  a different name like `test_wfirst_find_sca` where we only compare against Chris's SCA choices.  Because there are quite a few tests that got removed by the change here, which would be nice to be able to bring back eventually.

Reply:
Yes, that's a good idea.  To be honest it isn't clear when/if we'll get something to compare against from the project office (I've talked with Jeff about it but he is very busy).  Still,  doing what you suggest will serve as a reminder of what kinds of tests we may want to re-institute in some other way later on.  Will do in a sec.